app/src/views/Cart.vue

```diff
@@ -82,23 +86,42 @@
     </div>
 
     <!-- Checkout -->
-    <div class="mt-10">
-      <div>{{ cartSummaryString }}</div>
-      <div class="flex justify-center mt-5">
-        <button @click="checkout" class="btn btn-secondary">Checkout</button>
+    <div class="bg-white shadow sm:rounded-md max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 text-right">
+      <div class="border-b border-grey-100 py-8">
+        <span class="text-grey-300">Contributing:</span> {{ cartSummaryString }}
+      </div>
+      <div class="border-b border-grey-100 py-8"><span class="text-grey-300">Equivalent to:</span> TODO USD</div>
+      <div class="border-b border-grey-100 py-8">
+        <span class="text-grey-300">Estimated matching value:</span> TODO USD
```

Comment:
IMO we should. This PR was focused on UI work, and that involves some data work. IMO you or @thelostone-mc would be better equipped to handle these TODOs then I would since you're more familiar with `dcurve` than me